.storybook/recipes/GlobalHeader/GlobalHeader.tsx

```diff
@@ -173,9 +172,11 @@ export const GlobalHeader = ({
         />
       </Button>
 
-      <NavContainer
-        className={styles['global-header__nav-container']}
-        isActive={isActive}
+      <div
```

Comment:
nit: or `<section>` could work... or `<nav>`?

Reply:
`section` would not work since [MDN](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/section) suggests "Sections should always have a heading ... If you are only using the element as a styling wrapper, use a `<div>` instead"

The nested `PrimaryNav` component provides the `<nav>`